identity/backend/src/main/resources/application.yaml

```diff
@@ -1,13 +1,7 @@
 spring:
-  web:
-    resources:
-      cache:
-        cachecontrol:
-          max-age: 31536000
-          s-max-age: 31536000
-          must-revalidate: true
-          proxy-revalidate: true
-          no-transform: true
+  datasource:
+    username: ${CAMUNDA_IDENTITY_DATABASE_USERNAME:}
+    password: ${CAMUNDA_IDENTITY_DATABASE_PASSWORD:}
   jackson:
```

Comment:
@Ben-Sheppard, this configuration seems to configure a default Jackson object mapper. Did you ensure that this configuration does not conflict with the other components? @houssain-barouni can help to spot the places where the default object mapper is used.

Reply:
This configuration was here from an early iteration, as we don't need to do anything different/custom it seems to me like I can just remove this configuration. I'll try that first and then reach out if any issues come up